README.md

```diff
@@ -333,6 +331,8 @@ To specify Tenjin as the destination for your [SK Ad Network postbacks](https://
 
 These steps are adapted from Apple's instructions at [https://developer.apple.com/documentation/storekit/skadnetwork/configuring_an_advertised_app](https://developer.apple.com/documentation/storekit/skadnetwork/configuring_an_advertised_app).
 
+**NOTE**: If you are using AppLovin MAX for mediation, their SDK will overwrite Tenjin URL with their URL during the app build process. In that case, AppLovin can forward us the postbacks, so please contact your account manager to enable the forwarding.
```

Comment:
Small clean up:

"If you are using AppLovin MAX for mediation, their Unity SDK will overwrite any value you entered for `NSAdvertisingAttributionReportEndpoint` with their own URL during the build process. You should be able to set the `NSAdvertisingAttributionReportEndpoint` to `https://tenjin-skan.com` in XCode after its been overwritten, or you can ask your AppLovin account manager set up forwarding the postbacks to us."

Reply:
Thanks, updated.

Reply:
@amirmanji do you mind approving this?